NICE.Identity.Management/ClientApp/src/views/OrganisationsList/OrganisationsList.tsx

```diff
@@ -0,0 +1,451 @@
+		} = this.state;
+
+		const paginatePositions = this.getPaginateStartAndFinishPosition(
+			organisations.length,
+			pageNumber,
+			itemsPerPage,
+		);
+
+		const paginationText = this.getPaginationText(
+			organisations.length,
+			paginatePositions.start,
+			paginatePositions.finish,
+		);
+
+		const organisationsPaginated = organisations.length
+			? organisations.slice(paginatePositions.start, paginatePositions.finish)
+			: organisations;
+
+		const sortOptions = [
+			{ identifier: "alpha-asc", label: "A - Z" },
+			{ identifier: "alpha-desc", label: "Z - A" },
+			{ identifier: "date-asc", label: "Date ascending" },
+			{ identifier: "date-desc", label: "Date descending" },
+		];
+
+		return (
+			<>
+				<Breadcrumbs>
+					<Breadcrumb to="/overview" elementType={Link}>
+						Administration
+					</Breadcrumb>
+					<Breadcrumb>Organisations</Breadcrumb>
+				</Breadcrumbs>
+
+				<PageHeader heading="Organisations" className="page-header mb--d" />
+
+				{!error ? (
+					<Grid>
+						<GridItem cols={12}>
+							<Button
+								to={`/organisations/add`}
+								variant="cta"
+								elementType={Link}
+								className="mb--e"
+								data-qa-sel="add-organisation"
+							>
+								Add organisation
+							</Button>
+						</GridItem>
+						<GridItem cols={12} md={3}>
+							<FilterSearch
+								onInputChange={this.filterOrganisationsBySearch}
+								label={"Filter by organisation name"}
+							/>
+						</GridItem>
+						<GridItem cols={12} md={9} aria-busy={true}>
+							{isLoading ? (
+								<p>Loading...</p>
+							) : organisationsPaginated.length ? (
+								<>
+									<div className="text-right">
+										<SortOptions
+											options={sortOptions}
+											selectedIdentifier={this.state.sortedBy}
+											onSortClick={this.handleSortClick}
+										/>
+									</div>
+									<h2
+										className={styles.organisationsListSummary}
+										data-qa-sel="organisations-returned"
+									>
+										{paginationText}
+									</h2>
+									<ul
+										className="list--unstyled"
+										data-qa-sel="list-of-organisations"
+									>
+										{organisationsPaginated.map((organisation) => {
+											const { id, name, dateAdded } = organisation;
+											const organisationsListHeading = {
+												headingText: `${name}`,
+												link: {
+													elementType: Link,
+													destination: `/organisations/${id}`,
+												},
+											};
+
+											const organisationsListMetadata: Array<CardMetaData> = [
+												{
+													label: "Date created",
+													value: Moment(dateAdded).format("DD-MM-YYYY"),
+												},
+											];
+
+											return (
+												<li key={id}>
+													<Card
+														{...organisationsListHeading}
+														metadata={organisationsListMetadata}
+														key={id}
+													/>
+												</li>
+											);
+										})}
+									</ul>
+									<Pagination
```

Comment:
We have a fairly new `EnhancedPagination` component from NDS we might be able to use here if you're interested

Reply:
Yeah, I've copied all the pagination stuff here from something I did a while back for the UsersList component, it was before the nds had any pagination component and when I was probably a little more inexperienced with it all. I had to revisit some of it this time round as the sort options are new and noticed that some of it didn't make sense. Ideally I'd like to replace the implementation with design system component, but it was faster to just copy this over.